exercises/binary-search/canonical-data.json

```diff
@@ -4,11 +4,9 @@
     "The error object is used to indicate that the value is not included in the array.",
     "It should be replaced with the respective expression that is idiomatic",
     "for the language that implements the tests.",
-
```

Comment:
Should probably be replaced with a `"",` or ignored then because the newline seems intentional.